ocf_datapipes/transform/xarray/pv_power_rolling_window.py

```diff
@@ -0,0 +1,68 @@
+from typing import Optional, Union
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+from torchdata.datapipes import functional_datapipe
+from torchdata.datapipes.iter import IterDataPipe
+
+
+@functional_datapipe("pv_power_rolling_window")
+class PVPowerRollingWindowIterDataPipe(IterDataPipe):
```

Comment:
what does this do? get some pv power over a window? or average it over a window?

Reply:
It gets the mean over a rolling window for the PV power, again one of the the PP xr_batch_processors